src/graphics/image.js

```diff
@@ -37,6 +37,7 @@ Crafty.c("Image", {
 
     init: function () {
         this.bind("Draw", this._drawImage);
+        this.bind("LayerAttached", this._setupImage);
```

Comment:
How is image setup if already attached to layer?
```js
if (this._drawLayer) {
    this._setupImage(this._drawLayer);
}
```

maybe add this event like in `Color`:
```js
events: {
   "LayerAttached": "_setupImage"
}
```